backend/utils/cronUtils.js

```diff
@@ -0,0 +1,36 @@
+const MAX_CRON_MINUTE = 59;
+
+const ONE_MINUTE = 60;
+const FIVE_MINUTES = 300;
+const ONE_HOUR = 3600;
+const ONE_DAY = 86400;
+
+export function mapRefreshToCron (refreshInterval) {
```

Comment:
There are still cases for which this function returns suboptimal results; e.g.
```
mapRefreshToCron(ONE_MINUTE + ONE_SECOND) -> FIVE_MINUTES
mapRefreshToCron(FIVE_MINUTES + ONE_SECOND) -> ONE_HOUR
mapRefreshToCron(ONE_HOUR + ONE_SECOND) -> ONE_DAY
mapRefreshToCron(ONE_DAY + ONE_SECOND) -> ONE_WEEK
```

I'm going to add some comments to `getInitialCronMode` in https://github.com/plotly/falcon-sql-client/blob/798017a391be6d238a94235f0ad02bcdba1f3d3b/app/components/Settings/cron-picker/cron-helpers.js#L32-L45 to explain the choice of limits in the `if`'s.